client/stylesheets/_toCat.scss

```diff
@@ -3302,6 +3310,18 @@ On click the images will be displayed at normal size to complete the effect
     float: right;
 } 
 
+.listingItemLocationString {
+    margin-top: 18px;
+    padding: 10px 0px 10px 0px;
+    background-color: #f8504b;
+    border-radius: 4px;
+}
+
+.listingItemLocationString p {
```

Comment:
Merge rule `.listingItemLocationString p` with rule on line 3313<br>Selector `listingItemLocationString` should be written in lowercase with hyphens